testsuite/features/step_definitions/retail_steps.rb

```diff
@@ -474,7 +474,7 @@ def compute_kiwi_profile_version(host)
   elsif host == 'sle15sp3_terminal'
     mac = $sle15sp3_terminal_mac
     mac = 'EE:EE:EE:00:00:06' if mac.nil?
-  elsif host.include? 'ubuntu'
+  elsif (host.include? 'deblike') || (host.include? 'debian11') || (host.include? 'ubuntu')
```

Comment:
can't you use the `db_host?(host)` function here like you've been using it for the rest of the code? :thinking:

Reply:
No, as you can see, `debian11` does not work like `debian9` nor `debian10`.

They changed the interface naming conventions from one version to the other.